packages/components/src/templates/accordion-group.stories.ts

```diff
@@ -88,29 +120,47 @@ export const NeutralBackground = {
 };
 
 export const PrimaryBackground = {
-  name: 'Primary Background',
+  name: 'Accordion Group with Primary-100 Background',
   render: () => html`
     <div class="bg-primary-100 p-8">
       <sd-accordion-group>
-        <sd-accordion summary="Shareholder Structure"
-          ><p>
+        <sd-accordion summary="Shareholder Structure">
+          <figure class="sd-media">
+            <img
+              class="w-full aspect-ratio"
+              src="./placeholders/images/collaboration.jpg"
+              alt="Two men in business attire sitting on a sofa in a modern office, smiling and looking at a laptop screen."
+            />
+          </figure>
+          <p class="sd-paragraph mt-4">
             Union Investment is the dedicated asset manager within the German cooperative financial network. All
             shareholders are also part of the network, which ensures a stable and long-term ownership structure. This
             allows us to maintain a sustainable business model and develop strong, long-term client relationships.
-          </p></sd-accordion
-        ><sd-accordion summary="Cooparative financial network"
-          ><p>Union Investment is the asset manager within the German cooperative financial Network.</p></sd-accordion
-        ><sd-accordion summary="Investment philosophy">
+          </p>
+        </sd-accordion>
+        <sd-accordion summary="Cooperative financial network">
+          <h3 class="sd-headline sd-headline--size-xl mb-4">
+            Union Investment is the asset manager within the German cooperative financial Network.
+          </h3>
+          <figure class="sd-media">
+            <img
+              class="w-full aspect-ratio"
+              src="./placeholders/images/coins.jpg"
+              alt="Close-up of hands stacking coins into small piles on a table, suggesting financial planning or saving."
+            />
+          </figure>
+        </sd-accordion>
+        <sd-accordion summary="Investment philosophy">
           <div class="sd-prose">
-            <p>
+            <p class="sd-paragraph">
```

Comment:
Why do you need sd-paragraph here, when there is already sd-prose? Maybe it's more that we need to remove the Storybook styles e. g. in a decorator somewhere?

Reply:
Updated